src/content-items/serialization.ts

```diff
@@ -0,0 +1,23 @@
+import { RawDirectoryMetadata, RawFileMetadata } from '../pod/types'
+
+/**
+ * Serializable `FileItem`
+ */
+export interface FileItemSerializable {
```

Comment:
This file can be deleted as well?

Reply:
Sorry, I was confused by the classes names. Do we need the `Serializable` suffix? It might confuse regular users.